backend/schema.js

```diff
@@ -268,7 +269,7 @@ type Mutation {
     createJudge(input: UserInput!): User
     createAdmin(input: UserInput!): User
     updatePermissions(input: PermissionInput!): User
-    updateUser(id: ID!, input: UserInput!): User
```

Comment:
I don't think update user gets called anywhere else but I'll have to double check to make sure I didn't break anything

Reply:
tests all pass so this should be fine